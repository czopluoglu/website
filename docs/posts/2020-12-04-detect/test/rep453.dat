00101000111101101101
10000110110001000010
10101101100010011011
11000001111011010100
01000001011001011010
10111101010110000001
11011101110010100111
00010000111110111000
10000000011111100011
11111101000011111111
10100010101010001111
01100100111111111110
11110111111100100010
11010000100100101111
11111011111111010111
11100011101010100111
11000101000111111011
11000001111000111011
00011001001111010011
10001001001111101101
11110111111111111111
10001001111111101111
01011101000111011110
01001100101011101110
11101001110011011111
01001011001010101101
10010101010011000010
10100100011111011111
00001001001101111111
00000001100000100111
01001001011010000000
01001000101011000110
11001001010010111010
01000000011011110111
10111101101111011111
01111000101110101111
00111011010100110001
01111111110001100000
10001001111110110100
00111001001110111110
10011010111110111010
11111111101111100110
01100111011111110101
10001110010000100011
10011101001011101000
10000001001010100000
01011111011011110111
10101100000000000010
11110111011011101110
11111111111000111101
01100101110110000110
11010111111010101110
10000100110000001100
10111111111010101111
00010000101000111011
00000001001001111111
10000000011101011111
11000101001110110011
11000001111111111110
11000101001010011111
00010000001111011111
11010010101010101100
01010001001100000110
11000010001010110111
10000110100100110010
10000101010000100010
10111101111100001111
11000000011011111110
10100001111010110001
10000100111011101111
10001001101011000111
10110001110010001100
11001010100111100010
00000011111111111111
01010001111110001111
11111101101111111111
10000000001011101111
01000101110111111111
00011101110110000011
11001100010000110001
10000101111000101001
10111011111001100111
01100000010110101110
10101100100110010110
00111101100110001111
11010111110111111100
01000111011110001110
01001010001111111111
11100001001111110111
00100010011000110010
11110000011000011110
11000111100000110010
10100001011111111111
01001110111101101110
10000001111111110111
11010101000010101110
00001001011010101101
01101001111111100101
00001001000000100111
10111101111011001100
10011011111100000110
00111001011101101111
10111000110000010111
00000000001110101000
00110100011001001111
00010101111010101100
10110111001010101110
10100100011110001110
00110111100010010100
11001011111111001000
00011001111001110110
10110010100101101111
11111101011110100100
11111011001000101110
00100110101001111111
11011110111001111111
01101100001110100111
10101110101111111111
01111111001111111111
11111101111111111111
11001001001110001011
11110000111000100110
10100011101010101100
10000101110010101000
00100000001010100010
11001001011010000010
10001111011011111010
00001111111110011101
10000001001000001010
00000100001001000010
10000011111011101111
10100001110110000010
01100001001100000111
11100101111000101111
00001010011001111110
11011001111011100110
01001001110110100111
10011011011101111011
11110111111110111111
11010001100011010001
10000111100010001010
11011101101011011010
01000010111011010010
11000000000110101110
10011111111111100111
10111000001011000010
01111001110111011110
11110111110010100110
11010111011010111110
00011111100000110011
01000101110010111111
01000111011010011111
10110000011111111111
10011000101000100010
11001010111011111010
11111101110010111111
11101000000100000010
00101101000100000010
01111101110101110001
00001101111010000110
11010000111011101110
00111111111101101011
00101101001110011010
10011000100000001010
01110000111110110010
01011001011010001110
11111111110111010101
10110111111011100100
11100110100101101111
11101110101101111111
11000111011010101111
11101111101011111111
11000000101101100011
01010111001110100110
00011001110010101110
10101011110011011000
01000000101010111101
00010000000000100111
11011110101110101111
11011100100100000010
00011001001011011110
10011111110011101100
11101000101000000111
01010001011010110101
00000110100100000010
00000000011011110111
00010001000011100111
11101001101111011011
01010001001101100110
01100100001011101010
11010010011101111111
11011101111111011110
10000000100010100111
00100001011101100011
10100100011110101110
00000000111110111110
01100111111010100000
11010110110000110111
10111111111011110110
00010100111111010111
00011101011110111110
11001001011110111111
00010000001001000010
00111100001111111011
10101000010101011011
10111111110011100110
10101001001110001110
00110001100110110011
01001111110111111111
00101101111001101100
10000000001000001010
11001011011111110110
11100001101010101101
10010101001111101110
10111010011001000111
00010100100110111011
10111111111010111110
10001011101010100011
01111101010101111111
10110001011110111110
11001010111000000111
00001001101010100010
01001100011110011011
11111001001010111100
11101100001011110111
11100101111010111111
11010100000000100010
01001000110000100010
10001001100101001110
00010011001001111111
10111111001111011011
01011101111001000110
00001010000111101110
11000001111100101111
00011001011010000010
01011100111010100111
10011000111011010000
01001011010011110011
01111101111011011011
11110111110010100111
11011100001010101010
11111011110000100011
11001111111010111110
10100011011110010011
11101001010000011010
11010101011011111111
11001111111110101110
10101011011000101100
01111101001101101010
11010000110111001111
