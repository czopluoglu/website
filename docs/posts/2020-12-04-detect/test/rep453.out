-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep453.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.80

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.87
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.9216
                   IDN index value:   0.7684
                           Ratio r:   0.6862

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    3    3    3    3    1    3    1    1
    2    4    2    4    2    4    2    2    4    4

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    7    9   10

   -----------CLUSTER  2-------------

    2   11   13   15   17   18

   -----------CLUSTER  3-------------

    3    4    5    6    8

   -----------CLUSTER  4-------------

   12   14   16   19   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d++++-+-+-+---------
   +d++---+-----+------
   ++d++----+-+++------
   +++d++---+-+-++--+--
   +-++d-++-+---+---++-
   ---+-d++++------+---
   +---++d-++---+++-+--
   -+--++-d++---------+
   +----+++d+---+-+++--
   --+++++++d-+----+-++
   +---------d+++++---+
   --++-----++d+++-+---
   --+-------++d-+++---
   -++++-+-+-++-d+-----
   ---+--+---++++d-----
   ------+-+-+-+--d++++
   -----+--++-++--+d+++
   ---++-+-+------++d++
   ----+----+-----+++d+
   -------+-++----++++d
 
 No cross validation for this DETECT run
